docker-compose.yml

```diff
@@ -135,8 +135,8 @@ services:
       - fn-admin-storage
     labels:
       - "traefik.enable=true"
-      - "traefik.http.routers.functions.rule=Host(`localhost`)"
-      - "traefik.http.routers.functions.entrypoints=web"
+      - "traefik.http.routers.functions-admin.rule=Host(`functions-admin.localhost`)"
```

Comment:
why Host(`functions-admin.localhost`) ? does this require to change /etc/hosts ? in case, that would be better to discriminate only on ports (not on hostnames) to avoid it, you can use somthing like:

 - "traefik.http.services.functions.loadbalancer.server.port=7071"

Reply:
No, no changes to /etc/hosts are required with such configuration.